tests/basic_map_operations_test.py

```diff
@@ -8,16 +8,6 @@ class TestBasicMapOperations(unittest.TestCase):
     def setUp(self):
         pass
 
-    def test_random_point(self):
```

Comment:
The method random_point() isn't used anymore so I removed it (and this test). It was an almost trivial method anyway.